Makefile

```diff
@@ -74,12 +74,12 @@ clean:
 check-licenses: check-licenses-go-mod check-licenses-npm
 
 check-licenses-go-mod:
-	go get github.com/google/go-licenses
+	go install github.com/google/go-licenses@latest
```

Comment:
can you use $(GOBIN) instead of calling go directly

Reply:
Assume you meant $(GOCMD)